server/src/main/resources/rules/rules.js

```diff
@@ -1,4 +1,4 @@
-// Version: 5.21
+// Version: 5.22
```

Comment:
This is going to conflict with my rules changes as well. Depending on which gets merged first, one of us will have to go back and change this to 5.23. If yours win, remind me to update my PR.